lib/Util/License/Features/Subscribers.php

```diff
@@ -32,22 +34,50 @@ public function check() {
   }
 
   public function hasAPIKey() {
```

Comment:
I would consider renaming this to `hasValidApiKey`. I think this could be easily misinterpreted later by another developer because one could expect that `hasApiKey` covers also the situation when a user has an invalid API key in the database.  I see we pass the value to JS also as `has_api_key` so I think it would be safer to rename it as well.
What do you think?

Reply:
I agree, I will do that.